src/api/controllers/UserController.js

```diff
@@ -110,18 +113,19 @@ module.exports = {
           if (err.name === "ValidationError") {
             if (err.errors.email) {
               if (err.errors.email.kind === "regexp") {
-                res.status(500).json({ error: "Invalid email address" });
+                res.status(400).json({ error: "Invalid email address" });
               } else if (err.errors.email.kind === "unique") {
-                res.status(500).json({ error: "Email already exists" });
+                res.status(400).json({ error: "Email already exists" });
               }
             }
 
             if (err.errors.username) {
               if (err.errors.username.kind === "unique") {
-                res.status(500).json({ error: "Username already exists" });
+                res.status(400).json({ error: "Username already exists" });
               }
             }
-            res.status(500).json(err);
+            console.log(err);
+            res.status(400).json(err);
```

Comment:
This one should stay as 500 with no body. (and change the line above to console.error while you're at it)